src/game/behaviors/sparkle_spawn_star.inc.c

```diff
@@ -36,11 +36,15 @@ void set_sparkle_spawn_star_hitbox(void) {
     }
 }
 
-void set_home_to_mario(void) {
+void spawned_star_set_target_above_mario(void) {
```

Comment:
it is a more descriptive name but this also could make you think you can't use it for anything else. 

I think instead it should be `cur_obj_set_home_above_mario`

Reply:
oh wait and `o->oPosY = o->oHomeY;`
so `cur_obj_move_above_mario`?

Reply:
gdi I just realized which file this is in. just keep it lol

Reply:
beating a dead horse 😔

Reply:
i was already typing that before you sent your followup :pensive: